modules/sql-engine/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -473,67 +473,18 @@ SqlNodeList CreateZoneOptionList() :
 void CreateZoneOption(List<SqlNode> list) :
 {
     final Span s;
-    final SqlLiteral key;
+    final SqlIdentifier key;
     final SqlNode val;
 }
 {
-    (
-        (
-            key = CreateNumericZoneOptionKey() { s = span(); }
-            <EQ>
-            val = NumericLiteral()
-        )
-        |
-        (
-            key = CreateStringZoneOptionKey() { s = span(); }
-            <EQ>
-            val = StringLiteral()
-        )
-    )
+    key = SimpleIdentifier() { s = span(); }
+    <EQ>
+    val = Literal()
     {
         list.add(new IgniteSqlZoneOption(key, val, s.end(this)));
     }
 }
 
-SqlLiteral CreateNumericZoneOptionKey() :
-{
-}
-{
-    <PARTITIONS>
```

Comment:
also we need to remove all this keywords from config, because we don't need them anymore (keywords here are words enclosed in diamonds: PARTITIONS, REPLICAS, etc; see collections `keywords` and `nonReservedKeywords` in `modules/sql-engine/src/main/codegen/config.fmpp`)